Header header

string[] object_name
geometry_msgs/Point object_position
geometry_msgs/Quaternion object_orientation
geometry_msgs/Vector3 object_scale
